ansible/roles/ros_workspace/files/rosdep_install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source ${1}/devel/setup.bash
```

Comment:
I always found that using `{}` makes bash slightly less hard to read (identify what's the variable more clearly